src/main/routes/change-password-routes.ts

```diff
@@ -0,0 +1,8 @@
+import { type Router } from 'express'
+import { adaptRoute } from '@/main/adapters'
+import { makeChangePasswordController } from '@/main/factories'
+import { auth } from '@/main/middlewares'
+
+export default (router: Router): void => {
+  router.post('/change-password', auth, adaptRoute(makeChangePasswordController()))
```

Comment:
What do you think to use more semantic routes? example:
`post('/guardian/:id')` or  just `post('/guardian')` with query param id